docker-compose.yml

```
@@ -70,7 +70,7 @@ services:
         condition: service_healthy
 
   web:
-    image: python:3.10-slim-buster
+    image: python:3.12-slim-buster
```

```suggestion
    image: python:3.12-slim
```
There is no buster for 3.12 slim